Plugin/CppApi/source/CoordinateConversionController.cpp

```diff
@@ -262,39 +424,71 @@ void CoordinateConversionController::objectAppend(QQmlListProperty<QObject>* pro
   engine->addOption(option);
 }
 
+/*!
+  \brief Helper method to programtically add a CoordinateConversionOptions \a option
+  to the list of options.
+ */
 void CoordinateConversionController::addOption(CoordinateConversionOptions* option)
 {
   m_options.append(option);
   emit optionsChanged();
 }
 
+/*!
+  \brief Helper method to clear all the options from the list.
+ */
 void CoordinateConversionController::clearOptions()
 {
   m_options.clear();
   emit optionsChanged();
 }
 
+/*!
+  \brief Gets the name of this tool.
+ */
 QString CoordinateConversionController::toolName() const
 {
   return "CoordinateConversion";
 }
 
+/*!
+  \qmlmethod void CoordinateConversionController::copyToClipboard(string text)
+
+  \brief Helper method to copy \a text to the clipboard.
+ */
 void CoordinateConversionController::copyToClipboard(const QString& text)
 {
-  QApplication::clipboard()->setText(text);
+  auto clipboard = QApplication::clipboard();
```

Comment:
`QApplication` brings in Qt Widgets dependency.  Use `QGuiApplication::clipboard()` instead.